server/test/src/main/java/org/eclipse/lsp/cobol/test/engine/ExtensionContextCleaner.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.test.engine;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * ExtensionContextCleaner
+ */
+public class ExtensionContextCleaner implements AfterEachCallback {
+
+    /**
+     *
+     * @param context
+     */
+    @Override
+    public void afterEach(ExtensionContext context) {
```

Comment:
What is the reason to keep thise one method separate?

Reply:
I will place this under ExtensionContextProvider

Reply:
Placed it under ExtensionContextProvider. As this is  required only in test module